doc/storage/appendix/sra.rst

```diff
@@ -0,0 +1,919 @@
+
+   *  Read any input and output to the target through external apparatus.
+   *  Provide, forge, replay or modify such inputs and outputs.
+   *  Perform timings on the observable operations being done by the target, either in normal operation or as a response to crafted inputs. For example, timing attacks on web servers.
+
+:adversarial-model:`1`
+   The Adversary can additionally mount attacks from software running on a target processor implementing the feature. This type of Adversary can run software on the target.
+
+   For instance, the Adversary can:
+
+   *  Attempt software exploitation by running software on the target.
+   *  Exploit access to any memory mapped configuration, monitoring, debug register.
+   *  Mount any side channel analysis that relying on software-exposed built-in hardware features to perform physical unit and time measurements.
+   *  Perform software-induced glitching of resources such as Rowhammer, RASpberry or crashing the CPU by running intensive tasks.
+
+:adversarial-model:`2`
+   In addition to the above, the Adversary is capable of mounting hardware attacks and fault injection that does not require breaching the physical envelope of the chips. This type of Adversary has access to a system containing an implementation of the target feature.
+
+   For instance, the Adversary can:
+
+   *  Conduct side-channel analysis that requires measurement equipment. For example, this can utilize leakage sources such as EM emissions, power consumption, photonics emission, or acoustic channels.
+   *  Plug malicious hardware into an unmodified system.
+   *  Gain access to the internals of the target system and interpose the SoC or memory for the purposes of reading, blocking, replaying, and injecting transactions.
+   *  Replace or add chips on the motherboard.
+   *  Make simple, reversible modifications, to perform glitching.
+
+:adversarial-model:`3`
+   In addition to all the above, the Adversary is capable of performing invasive SoC attacks.
+
+   For instance, the Adversary can:
+
+   *  Decapsulate a chip, via laser or chemical etching, followed by microphotography to reverse engineer the chip.
+   *  Use a focussed ion beam microscope to perform gate level modification.
+
+The adversarial models that are in scope for storage depend on the product requirements. To ensure that the |API| can be used in a wide range of systems, this assessment considers adversarial models `AM.0`, `AM.1`, and `AM.2` to be in-scope.
+
+.. _usability:
+
+.. _sra-threats:
+
+Threats
+^^^^^^^
+
+Code in the RoT partitions is assumed to be trustworthy - and any untrustworthy code running in that partition already has complete control of the target - therefore in `AM.1` we only consider threats from malicious actors running in :term:`Non-secure Processing Environment`. 
+
+When considering threats, we transfer the risk of protecting different users within a single partition to the operating system or run time within that partition.
+

```

Comment:
There is a missing T.INTERFACE_ABUSE threat. For example see the FWU SRA: https://github.com/ARM-software/psa-api/blob/80ea7319d0e86fae6d4328cf28ee3d188461ae69/doc/fwu/appendix/sra.rst?plain=1#L626